src/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { User } from "./types"
+
+// ESLint is disabled here purely to help with clean code
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const anyToUser = (input: any): User => {
```

Comment:
This seems to be an identity function, I guess it's used because of types.
But, wouldn't it be better to just use typescript typecasts instead of this?